### ISSUE TITLE
PlayAnimation in Tasks

### ISSUE THREAD

**Issue by IncoCode** [crosire/scripthookvdotnet#55]
I think PlayAnimation method should be in Tasks too. Because playing animation may be a part of sequence:
```c#
var s = new TaskSequence();
s.AddTask.EnterVehicle( veh, VehicleSeat.Driver );
s.AddTask.DriveTo( veh, pos, 3f, 15 );
s.AddTask.LeaveVehicle();
s.AddTask.PlayAnimation( animSet, "say_hurry_up_a", 4f, 3, false, 0 );
s.AddTask.EnterVehicle( veh, VehicleSeat.Driver );
s.CloseSequence();

ped.Task.PerformSequence( s );
```

**Comment by crosire** [comment]
Makes sense, yeah.

**Comment by crosire** [comment]
Fixed in 0fb5d0611f230cb742bf60335727e7f7780b4d51.

**Comment by TheChilliPL** [comment]
How to use it? I can't find it anywhere...
